MediaManager/Plugin.MediaManager.Abstractions/IMediaFile.cs

```diff
@@ -15,11 +14,6 @@ namespace Plugin.MediaManager.Abstractions
     public interface IMediaFile
     {
         /// <summary>
```

Comment:
I agree on removing this, because an item is equal when the `Type` and `Url` matches.